go/libraries/doltcore/sqle/database_provider.go

```diff
@@ -279,6 +281,15 @@ func (p DoltDatabaseProvider) AllDatabases(ctx *sql.Context) (all []sql.Database
 			foundDatabase = true
```

Comment:
You might want to set `foundDatabase` in the `if showBranches` loop, too; otherwise, I think you get the revision db showing up in the list twice if you have this var set and you have `use`d a revision db in the current session.

Reply:
Very good catch, this was double-included revision dbs